### PR TITLE
Remove biases before Batch Normalization

### PR DESCRIPTION
**Pull request by mlopezantequera**
Biases are negated by the internal bias (mean) of batch normalization, they can be removed without any harm, saving some memory and compute effort.

I haven't tested this myself as I can't train for any decent length at the moment, so perhaps somebody can test it before merging.

### CODE REVIEW

**Comment by kwotsin** [comment]
I think they might be safe to be removed, but I'm not exactly sure whether this initialization is important for other ops deep within TF. For what it's worth, the bias regularizer option is None by default, as seen here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py

In the Keras code, it seems that the author also built it the same way, i.e. zeros are initialized but not really used. See here: https://github.com/fchollet/keras/blob/master/keras/applications/xception.py
